apps/inkscape/inkscape.txt

```diff
@@ -0,0 +1 @@
+the vector graphics editor
```

Comment:
Use Inkscape's description of their product: "Inkscape is an open-source vector graphics editor similar to Adobe Illustrator, Corel Draw, Freehand, or Xara X. What sets Inkscape apart is its use of Scalable Vector Graphics (SVG), an open XML-based W3C standard, as the native format."

Reply:
okie dokie...

Reply:
ok done...